### PR TITLE
Formating & Improvements

### PR DESCRIPTION
**Pull request by pgert**

### CODE REVIEW

File: patches/SLES-50672_4A9D8E01.pnach [hunk 1]
```diff
@@ -1,33 +1,75 @@
...
+patch=1,EE,0013D280,word,3C014464
+patch=1,EE,00152C40,word,3C014464
+// ==========
+patch=1,EE,003224C8,word,3FBF851F
+patch=1,EE,003224EC,word,3FBF851F
+// ==========
+
+[No-Interlacing]
+author=Agrippa
+patch=1,EE,00204C58,word,30420000
```
[the remark this answers is not in the document]
Comment (Gabominated) [reply patches/SLES-50672_4A9D8E01.pnach:75]: There isn't improvement here, this kind of no-interlacing code is not the solution

File: patches/SLPM-65845_C3B698F9.pnach [hunk 2]
```diff
@@ -1,41 +1,53 @@
...
+patch=1,EE,002c4c64,word,3c013ff0 // 3c013faa
+patch=1,EE,002c4c68,word,34210000 // 34213d71
+//patch=1,EE,002c0efc,word,3c013de3 // 3c013faa
+//patch=1,EE,002c0f00,word,3421f70a // 34213d71
+//patch=1,EE,002c4c64,word,3c013fe3 // 3c013faa
+//patch=1,EE,002c4c68,word,3421f70a // 34213d71
+
+[No-Interlacing]
+author=pcsx2user156 (ported by)
+patch=1,EE,001E7D08,word,30630000
```
[the remark this answers is not in the document]
Comment (Gabominated) [reply patches/SLPM-65845_C3B698F9.pnach:44]: No recommend, I thought that was clearly, this kind of codes make more damage

File: patches/SLES-50672_4A9D8E01.pnach (hunk 1 above)
[the remark this answers is not in the document]
Comment (pgert) [reply patches/SLES-50672_4A9D8E01.pnach]: Deinterlacing hack removed.

File: patches/SLPM-65845_C3B698F9.pnach (hunk 2 above)
[the remark this answers is not in the document]
Comment (pgert) [reply patches/SLPM-65845_C3B698F9.pnach]: Deinterlacing hack removed.

File: patches/SCES-50006_04F9D87F.pnach [hunk 3]
```diff
@@ -1,19 +1,18 @@
-gametitle=Drakan - The Ancients' Gates (PAL-M5) SCES-50006 04F9D87F
+gametitle=Drakan - The Ancients' Gates * PAL-M5 * SCES-50006 * 04F9D87F
 
-[Widescreen 16:9]
-gsaspectratio=16:9
-author=Dreadmoth and pgert
```
[the remark this answers is not in the document]
Comment (pgert) [reply patches/SCES-50006_04F9D87F.pnach:5]: Corrected.

**Comment by pgert** [comment]
No I haven't, and I can't (since I don't own them).
Shall I remove the hacks for those as well (3 pnachs more)?

**Comment by pgert** [comment]
Done.

**Comment by kamfretoz** [comment]
What's the status for this PR?

**Comment by pgert** [comment]
It's ready, but someone have to "squash" it for me.

**Comment by refractionpcsx2** [comment]
Can you remove the cheats information from the patches, please? Completely irrelevant.

**Comment by pgert** [comment]
Anything else?

**Comment by pgert** [comment]
New WS-hack for the PAL pnachs of Champions of Norrath by PeterDelta.
Tested SLES-52325_75D86958, and can't make out any difference from the old hack by ElHecht.
The hacks by PeterDelta has fever patches (4 instead of 13/14), and consequestly there is no point of combination with 'Widescreen 16:9' for the ported hacks - should be less confusing to end-user.
Added 16:10, 15:10, 20:9 & 21:9 ports.

**Comment by refractionpcsx2** [comment]
Would you not rather use the `author=` parameter instead of putting `//pgert` in the files? I'm sure that means less to people than having the author.

**Comment by pgert** [comment]
Is this better?

**Comment by pgert** [comment]
I thought it looked better.
Do you insist?

**Comment by pgert** [comment]
Done.

**Comment by pgert** [comment]
...?

**Comment by refractionpcsx2** [comment]
Sorry, forgot about it.

**Pull request merged**